src/error.js

```diff
@@ -374,6 +376,14 @@ export function getErrorReportUrl(message, filename, line, col, error,
   url += '&fr=' + encodeURIComponent(self.location.originalHash
       || self.location.hash);
 
+  // Google App Engine maximum URL length.
+  if (url.length >= 2072) {
+    url = url.substr(0, 2072 - 8 /* length of suffix */)
+        // Full remove last URL encoded entity.
+        .replace(/\%[^&%]+$/, '')
```

Comment:
Shouldn't this be removing all parts of the URL encoded entry? Ie, `encodeURIComponent('💩') = %F0%9F%92%A9`, so shouldn't we be searching for `/(\%[0-9a-fA-F]{2})+$/`?